packages/react-examples/src/react/Icon/Icon.Svg.Example.tsx

```diff
@@ -45,13 +52,13 @@ const BorderBlindsIcon: React.FunctionComponent<{ color1?: string; color2?: stri
 
   // FontIcon is an optimized variant of standard Icon.
   // You could also use the standard Icon here.
-  // Provide an `aria-label` (and set `role="img"`) for screen reader users if the icon is not accompanied by text
-  // that conveys the same meaning.
+  // This example demonstrates how to expose the SVG itself as an image with
+  // an accessible name. To do that, set aria-label or aria-labelledby directly
+  // on the SVG, and set aria-hidden to undefined or false on the FontIcon.
   return (
     <FontIcon
-      aria-label="Map icon"
-      role="img"
       iconName="borderblinds-svg"
+      aria-hidden={undefined}
```

Comment:
What is this meant to do? Generally it's best if we can avoid explicitly setting things to undefined, since it can easily be mixed up with the prop being unspecified depending on  exactly how the props object is used/accessed (which could also change over time).

Reply:
It was because the SVG handles the name itself, so I didn't want it to be hidden. I could set it to `aria-hidden="false"`, though that has the potential to cause problems if a parent sets `aria-hidden="true"` (i.e. the child could be interpreted to be shown if it's explicitly "false")